src/track/track.cpp

```diff
@@ -713,10 +713,10 @@ CuePointer Track::createAndAddCue() {
     return pCue;
 }
 
-CuePointer Track::findCueByType(Cue::Type type) const {
+CuePointer Track::findCueByType(mixxx::CueType type) const {
     // This method cannot be used for hotcues because there can be
     // multiple hotcues and this function returns only a single CuePointer.
-    DEBUG_ASSERT(type != Cue::Type::HotCue);
+    DEBUG_ASSERT(type != mixxx::CueType::HotCue);
```

Comment:
Unrelated: We should use VERIFY_OR_DEBUG_ASSERT and always return a nullptr in this case. Just to be sure that we never return any hot cue.